dask_expr/_collection.py

```diff
@@ -725,10 +741,8 @@ def idxmin(self, skipna=True, numeric_only=False):
     def idxmax(self, skipna=True, numeric_only=False):
         return new_collection(self.expr.idxmax(skipna, numeric_only))
 
-    def min(self, skipna=True, numeric_only=False, min_count=0, split_every=False):
-        return new_collection(
-            self.expr.min(skipna, numeric_only, min_count, split_every)
-        )
+    def min(self, skipna=True, numeric_only=False, split_every=False):
+        return new_collection(self.expr.min(skipna, numeric_only, split_every))
```

Comment:
We should make those consistent before releasing finally, but fixing tests has higher priority now. Someone has to go over those to make the argument order consistent at some point

dask/dask uses None because it supports pandas < 2 where the default was None as well, pandas changed the default to False in 2.0, so that's what we can use here as well to avoid handling logic around the default value

Reply:
gotcha, thanks! :+1: